doc/api/fs.md

```diff
@@ -685,6 +685,33 @@ added: v10.0.0
 
 Changes the ownership of a file.
 
+### `fsPromises.copy(src, dest[, options])`
```

Comment:
The name and options of `fs.rm` were chosen to be close to Unix `rm` command. Was it considered here?

Reply:
This was why I opted for `force` rather than `overwrite`, based on:

https://man7.org/linux/man-pages/man1/cp.1.html

`dereference` is also taken from `cp`, as is `preserve` (although I think it's worth including the `Timestamps` suffix for clarity).

`filter` and `errorOnExist` didn't really have an equivalent, but I thought this was okay.

Reply:
My question is mainly about the name of the function itself. It could be `cp` instead of `copy` (that would also solve the conflict raised in https://github.com/jprichardson/node-fs-extra/issues/912)

Reply:
@targos I like that this would be consistent with `rm`, but I'm concerned it would deviate form the existing `copyFile` method.

Reply:
@targos if we were to make it `cp`, I think we would be smart to go back to `overwrite` defaulting to `true`, as this behavior deviates from `cp` (`cp` overwrites an existing file by default I believe).

Reply:
I don't have strong feelings about this, TBH. You could also say that Unix `cp` isn't recursive by default.
I'll accept whatever the tooling team agrees on.

Reply:
I could see `cp` helping to differentiate us from the existing `copyFile`, which might be a good thing.

@targos @iansu @aduh95 @RyanZim what if we did the following:

* switch to calling this `cp` and `cpSync`.
* introduce the `recursive` option, which defaults to `false`.
  * if recursive is `false`, we just run the current `copyFile`.
  * if recursive is `true`, we use the new `copy` method.
* switch back to `force` being `true`, which is closer to the behavior of `cp`.

Reply:
I've pushed a change with these udpates.